specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/virtualNetworkGateway.json

```diff
@@ -4284,8 +4284,7 @@
         "ECP256",
         "ECP384",
         "PFS24",
-        "PFS14",
-        "PFSMM"
+        "PFS14"
```

Comment:
>PFS14 [](http://example.com/codeflow?start=9&length=5)

is removing PSMM intended? non-breaking?

Reply:
Yes, it is intended. This merged PR : [virtualNetworkGateway.json](https://github.com/Azure/azure-rest-api-specs/pull/29993/files/7432b5df05cb85550da88da3d8932ae539d5c816#diff-1735ce4b05b4eb68f09c6d8e95a0a17a8605b2b3fd65114df74bba614f39ec05) introduced the changes